Game.cpp

```diff
@@ -0,0 +1,146 @@
+#include "Game.h"
+#include "Map.h"
+
+void Game::setMap(Map map)
+{
+    
+}
+void Game::putHero(Hero hero,int x,int y)
+{
+    if(map.get(x,y) == Map::Free)
+    {
+        hero_location = std::make_pair(x+1,y);
+        this->hero = new Hero(hero);
+    }
+    else throw Game::OccupiedException("Location is not avaibile! \n");
+}
+void Game::putMonster(Monster monster,int x, int y)

```

Comment:
a map beállítás vizsgálata ebből is hiányzik

Reply:
done